core/src/main/java/org/elasticsearch/plugins/RemovePluginCommand.java

```diff
@@ -56,7 +56,18 @@ protected void execute(Terminal terminal, OptionSet options, Environment env) th
         execute(terminal, arg, env);
     }
 
-    // pkg private for testing
+    /**
+    * Execute on the given terminal and environment method, with pluginName derived by the argument options.
+    * When successful it removes a plugin from elasticsearch.
+    *
+    * @param terminal the Terminal to use for input/output
+    * @param pluginName the name of the plugin, given by the options
+    * @param env the environment for the local node, which may be used for the local settings and path
+    *
+    * @throws UserException if plugin name is null
```

Comment:
While you are here it is probably worth mentioning that this will print the the command's usage info. The other might not, I didn't scroll down and check because I'm on mobile.